integrations/terraform-cloud/main.py

```diff
@@ -165,8 +164,21 @@ async def resync_state_versions(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
         yield enriched_state_versions_batch
 
 
+@ocean.on_resync()
+async def on_create_webhook_resync(kind) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    terraform_client = init_terraform_client()
+    await terraform_client.create_workspace_webhook(
+        app_host=ocean.integration_config["app_host"]
```

Comment:
You first need to check if `app_host` is provided before you execute this. Otherwise it will throw an KeyError